src/optimum/intel/neural_compressor/utils.py

```diff
@@ -34,8 +34,6 @@ def from_pytorch_dataloader(cls, dataloader: DataLoader):
 
     def __iter__(self):
         for input in super().__iter__():
-            if not isinstance(input, (dict, tuple, list)):
-                raise TypeError(f"Model calibration cannot use input of type {type(input)}.")
```

Comment:
I delete them because model calibration cannot use input of type <class 'transformers.tokenization_utils_base.BatchEncoding'> in summarization task.